src/kaleidoscope/hardware/kbdfans/KBD4x.h

```diff
@@ -24,9 +24,19 @@
 #include "Kaleidoscope-HIDAdaptor-KeyboardioHID.h"
 
 #include "kaleidoscope/macro_helpers.h"
-#include "kaleidoscope/hardware/avr/pins_and_ports.h"
+
+struct cRGB {
+  uint8_t g;
+  uint8_t r;
+  uint8_t b;
+};
+
+#define CRGB(r, g, b) (cRGB){g, r, b}
```

Comment:
I feel like this should be a separate PR -- it's adding color support to the 4x, not really adding underglow to kaleidoscope

Reply:
The reason I opted to add color (well, underglow) support to kbd4x in the same PR is because UnderglowControl relies on the hardware parts to work, and it felt weird having a dependency without anything to satisfy it.

I can lift the KBD4x part out with little effort though, so I'll do that.